cmd/kubeadm/app/util/config/cluster.go

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package config provides the configuration file for
+// loading the kubeadm subcommand and setting the defaults for them.
```

Comment:
i think we can remove this comment?

Reply:
The ci does not seem to pass without this comment, because golangci-lint requires that there must be a package comment under a package, this description may not be very accurate, do you have any other suggestions?

Reply:
ok, let's leave this comment:
> // Package config contains utilities for managing the kubeadm configuration API.

Reply:
hmm, but if it's required here why it's not required in the new file cmd/kubeadm/app/util/config/upgradeconfiguration.go
?
that's odd.

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/123068/pull-kubernetes-linter-hints/1758033963663757312

I tried to delete the comment, and this is the error message. 
ok, I'm going to ask on slack.

Reply:
ok, I move it to common.go.